var/spack/repos/builtin/packages/cscope/package.py

```diff
@@ -10,8 +10,15 @@ class Cscope(AutotoolsPackage):
     """Cscope is a developer's tool for browsing source code."""
 
     homepage = "http://cscope.sourceforge.net/"
-    url      = "http://downloads.sourceforge.net/project/cscope/cscope/15.8b/cscope-15.8b.tar.gz"
```

Comment:
You should keep this URL here (or update it to the new URL), as it is used by `spack versions` and `spack checksum` to locate new versions. Although this is buggy on SourceForge, because they don't like external scripts crawling their webpages.